packages/contract-wrappers/src/coordinator_wrapper.ts

```diff
@@ -295,82 +263,6 @@ export class CoordinatorWrapper {
         return txHash;
     }
 
-    /**
-     * Synchronously executes multiple calls to fillOrder until total amount of makerAsset is bought by taker.
-     * Under-the-hood, this method uses the `feeRecipientAddress`s of the orders to looks up the coordinator server endpoints
-     * registered in the coordinator registry contract. It requests a signature from each coordinator server before
-     * submitting the orders and signatures as a 0x transaction to the coordinator extension contract, which validates the
-     * signatures and then fills the order through the Exchange contract.
-     * If any `feeRecipientAddress` in the batch is not registered to a coordinator server, the whole batch fails.
-     * @param   signedOrders         An array of signed orders to fill.
-     * @param   makerAssetFillAmount Maker asset fill amount.
-     * @param   takerAddress         The user Ethereum address who would like to fill these orders. Must be available via the supplied
-     *                               Provider provided at instantiation.
-     * @param   orderTransactionOpts Optional arguments this method accepts.
-     * @return  Transaction hash.
-     */
-    @decorators.asyncZeroExErrorHandler
-    public async marketBuyOrdersAsync(
```

Comment:
There are new methods around similar functionality, we should add a TODO in a back log to add these in as they are relatively useful.

Reply:
https://github.com/0xProject/0x-monorepo/issues/2189 ty!